src/index.html

```diff
@@ -81,7 +83,209 @@ <h3>I'm a Software Developer</h3>
         </div>
       </div>
     </section>
+
+    <!-- headline-container end -->
+
+    <!-- My Recent Works section start -->
+    <section class="section-works">
+      <article class="Recent-work">
+        <h2 class="works-title">My Recent Works</h2>
+        <img class="feature-image" src="./images/background-2.svg" alt="feature image about works">
+        <h3 class="title-post">Multi-Post Stories</h3>
+        <p class="post-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+          dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="button">
+          <li>
+            <button type="button" autofocus>CSS</button>
+          </li>
+          <li>
+            <button type="button" autofocus>Html </button>
+          </li>
+          <li>
+            <button type="button" autofocus>Bootstrap</button>
+          </li>
+          <li>
+            <button type="button" autofocus>Ruby</button>
+          </li>
+        </ul>
+        <button class="see-project-btn" type="button" autofocus>
+          See Project
+        </button>
+      </article>
+      <div class="work-cards">
+        <article class="work-card-1 card">
+          <h3>Profesional Art <br>
+            Printing Data</h3>
+          <p class="description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+            standard
+          </p>
+          <ul class="button-list button">
+            <li>
+              <button type="button" autofocus>Html</button>
+            </li>
+            <li>
+              <button type="button" autofocus>Bootstrap </button>
+            </li>
+            <li>
+              <button type="button" autofocus>Ruby</button>
```

Comment:
- [ ] Just as I mentioned before, kindly remove the button tags since they are list items and not buttons. So you can have something like this:
```
<ul>
  <li>CSS</li>
  <li>HTML</li>
  <li>Bootstrap</li>
  <li>Ruby</li>
 </ul>
```
- [ ] Please repeat this in all the other work cards :+1:

Reply:
resolved